prog/postgres/postgres_server_nexus.rb

```diff
@@ -54,11 +54,7 @@ def self.assemble(resource_id:, timeline_id:, timeline_access:, representative_a
 
   def before_run
     when_destroy_set? do
-      should_destroy = if ["destroy", nil].include?(postgres_server.resource&.strand&.label)
-        true
-      else
-        !(@snap.set?(:take_over) || ["prepare_for_take_over", "taking_over"].include?(strand.label))
-      end
+      should_destroy = ["destroy", nil].include?(postgres_server.resource&.strand&.label) || !(@snap.set?(:take_over) || ["prepare_for_take_over", "taking_over"].include?(strand.label))
```

Comment:
@byucesoy What do you think?

Reply:
I'm OK with the change, though I think we should put each of the three conditions on a separate line.

Reply:
I changed it
https://github.com/ubicloud/ubicloud/pull/3046/commits/5ca08d6f731858188142571cace1040d74c4a5d1

Reply:
Your change looks good :+1: